lib/pbench/server/auth/__init__.py

```diff
@@ -30,16 +24,22 @@ class KeycloakOpenID:
     :param headers: dict of custom header to pass to each HTML request
     """
 
+    AUTHORIZATION_ENDPOINT: str = ""
+    TOKEN_ENDPOINT: str = ""
+    USERINFO_ENDPOINT: str = ""
+    REVOCATION_ENDPOINT: str = ""
+    JWKS_URI: str = ""
+    LOGOUT_ENDPOINT: str = ""
```

Comment:
These aren't *CONSTANTS* if you're changing them after class creation; they should be lowercase.

I also suggest encapsulating the mechanism that populates them rather than having external code doing something like

```
        if not OpenIDClient.AUTHORIZATION_ENDPOINT:
            oidc_client.set_well_known_endpoints()
```

Reply:
I mean they are supposed to be constant in the sense that they are fixed by the respective identity provider. If we create an `oidc_client` for google then these variables should not be changed for that client object which them constant I guess.